_posts/2019-03-01-preparing-the-bagwell-award-2019.md

```diff
@@ -0,0 +1,31 @@
+---
+category: blog
+title: "Preparing the 2019 Phil Bagwell Community Award"
+by: Josh Suereth
+---
+
+Hello Scala Community!
+
+It is almost Scala Days and that means it is time to select Phil Bagwell award winners!
+This year, we are taking the time to formalize the rubric used to select winners, as well as make it more obvious who makes the decisions.
+
+We have drafted [the formal rubric here](https://www.scala-lang.org/community/phil-bagwell-award.html), for all to read.
+The essential gist is:
+
+* We recognize people who make us proud to be part of the community
+* Winners should contribute their time in one or more of the following:
+  * Technical Contribution
+  * Activism
+  * Evangelism
+
+Additionally, the committee to select award winners will consist of Martin Odersky and all previous winners of the award (who have time to take part).
+
+The selection process will first involve soliciting nominations from the community.
+Please let us know who you think meets [the published rubric](https://www.scala-lang.org/community/phil-bagwell-award.html) and why.
+The committee will hold voting later in March to select the 2018 and 2019 winners and they will be announced at Scala Days 2019 in Lausanne.
+
+So, please forward all submissions to TODO!
```

Comment:
add link in place of TODO

Reply:
See my question to Josh in the PR description. ;)

Reply:
ah, right, we need an actual address, sorry about that

Reply:
SO I had that in my original CL: https://github.com/jsuereth/scala-lang/commit/15d91131e3cee357061f1033f2eaf4654d0a5458

Reply:
bagwellaward@gmail.com

Reply:
Darja made a google form though, that's probably a better way to take in submissions.